src/main/java/org/sciborgs1155/lib/SparkUtils.java

```diff
@@ -108,4 +109,23 @@ public static void configureFrameStrategy(
   public static void configureFollowerFrameStrategy(CANSparkBase spark) {
     configureFrameStrategy(spark, Set.of(), Set.of(), false);
   }
+
+  /**
+   * Configures a CANSpark motor.
+   *
+   * @param motor The motor object. This is either a CANSparkMax object or a CANSparkFlex object.
+   * @param inverted The state of inversion. True if inverted.
+   * @param idleMode Idle mode setting (either COAST or BRAKE).
+   * @param limit Current limit in Amps.
+   * @return The motor object.
+   */
+  public static CANSparkBase configureMotor(
+      CANSparkBase motor, boolean inverted, IdleMode idleMode, int limit) {
+    motor.restoreFactoryDefaults();
+    motor.setInverted(inverted);
+    motor.setIdleMode(idleMode);
+    motor.setSmartCurrentLimit(limit);
```

Comment:
for the limit, another option is you can take in a Measure<Current> instead of a double. Then you can do the casting in here, and the code will be slightly less repetitive/verbose